notebooks/270-sound-generation/270-sound-generation.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        ov_model.inputs[0].get_node().set_partial_shape(ov.PartialShape((2, 8, -1, 16)))
</pre><p>Can we specify shapes in <code>input</code> parameter of <code>convert_model</code> function?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
no, this step is required for SD UNet to work. It is there in all our SD notebooks